### PR TITLE
docs: update SvelteKit instructions

### PR DESCRIPTION
**Pull request by benmccann**
### Description

That was the old way to configure it. We now put Vite config in `vite.config.js` instead of `svelte.config.js`

### CODE REVIEW

**Comment by userquin** [comment]
Thx Ben, I updated the example but not the docs...

File: README.md [hunk 1]
```diff
@@ -188,32 +188,25 @@ module.exports = {
...
+
+The `unplugin-icons` plugin should be configured in the `vite.config.js` configuration file:
 
 ```ts
-// svelte.config.js
-import adapter from '@sveltejs/adapter-auto'
-import preprocess from 'svelte-preprocess'
+// vite.config.js
+import { defineConfig } from 'vite'
+import { svelte } from '@sveltejs/kit/vite'
```
Comment (userquin) [inline README.md:198]: Shoukd be `sveltekit`

File: README.md [hunk 2]
```diff
@@ -188,32 +188,25 @@ module.exports = {
...
-export default config
+export default defineConfig({
+  plugins: [
+    sveltekit(),
+    Icons({
+      compiler: 'svelte',
+    })
+  ]
+})
+```
```
Comment (userquin) [inline README.md:209]: Remove extra backticks

Comment (benmccann) [inline README.md:209]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -188,32 +188,24 @@ module.exports = {
...
+
+The `unplugin-icons` plugin should be configured in the `vite.config.js` configuration file:
 
 ```ts
-// svelte.config.js
-import adapter from '@sveltejs/adapter-auto'
-import preprocess from 'svelte-preprocess'
+// vite.config.js
+import { defineConfig } from 'vite'
+import { svelte } from '@sveltejs/kit/vite'
```
Comment (benmccann) [inline README.md:198]:
```suggestion
import { sveltekit } from '@sveltejs/kit/vite'
```

File: README.md (hunk 2 above)
Comment (benmccann) [reply README.md:209]: done

File: README.md (hunk 1 above)
Comment (benmccann) [reply README.md]: yes, thanks! done

**Pull request merged**